misc/AppImageBuilder.yml

```diff
@@ -1,27 +1,25 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
 version: 1
-
 AppDir:
-  path: ./AppDir
-
+  path: AppDir
   app_info:
     id: net.minetest.minetest
     name: Minetest
     icon: minetest
     version: !ENV ${VERSION}
-    exec: usr/bin/minetest
+    exec: usr/local/bin/minetest
     exec_args: $@
-  runtime:
-    env:
-      APPDIR_LIBRARY_PATH: $APPDIR/usr/lib/x86_64-linux-gnu
-
+  files:
+    include: []
+    exclude:
+    - usr/share/man
+    - usr/share/doc
+    - usr/share/lintian
   apt:
     arch: amd64
     sources:
-      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal main universe
-        key_url: 'http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3b4fe6acc0b21f32'
-      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal-updates main universe
-      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal-backports main universe
-      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal-security main universe
+      - sourceline: 'deb http://archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse'
```

Comment:
afaik if we bump these it will mean the appimage won't work on any distro with libraries older than ubuntu 22.04
if this is true we might event want to _downgrade_ to 18.04

Reply:
@sfan5 I have no idea what your talking about?
![Screenshot from 2023-01-05 12-27-38](https://user-images.githubusercontent.com/96554164/210873830-b6d13f0d-d700-4ab9-aeec-56d849938866.png)

Reply:
https://docs.appimage.org/reference/best-practices.html#binaries-compiled-on-old-enough-base-system

Reply:
Well somehow I got the appimage to work with Jammy sources in Bionic. This is probably because I've included libc6.

Reply:
But if that is the case then I guess I can downgrade the AppImage to Bionic then.

Reply:
If you don't mind the work please downgrade it to bionic and take care to reuse system libraries (as the best practices say).

Reply:
@sfan5 you cannot build with Bionic. This is because (at least the first issue) in `src/script/lua_api/l_util.cpp` uses `ZSTD_CLEVEL_DEFAULT` which Bionic does not have because their Zstd is very outdated. I can however Mintest build with Focal. However when running (at least for me) I get a segmentfault in `porting::initializePaths()`, in `src/main.cpp`.

Reply:
Regarding bionic that's weird since we have working CI for that: https://github.com/minetest/minetest/actions/runs/3877576373/jobs/6612745670

Reply:
@sfan5 Well because I can't seem to build it but you can, I'm just going to revert my changes there then since I can't test changing it to Bionic and the fact that Bionic is soon going to be EOL.